spec/classes/profile/solr_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+# Copyright (c) 2018-2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+require 'spec_helper'
+
+describe 'nebula::profile::solr' do
+  on_supported_os.each do |os, os_facts|
+    context "on #{os}" do
+      let(:facts) { os_facts }
+
+      it { is_expected.to compile }
+
+      context 'with package' do
+        [
+          'openjdk-8-jre-headless',
+          'solr',
+          'lsof',
+        ].each do |package|
+          it { is_expected.to contain_package(package) }
+        end
+      end
+
+      context 'with directory' do
+        [
+          '/var/lib/solr',
+          '/var/lib/solr/home',
+          '/var/lib/solr/logs',
+        ].each do |path|
+          it do
+            is_expected.to contain_file(path).with(
+              owner: 'solr',
+              group: 'solr',
+              ensure: 'directory',
+              mode: '0750',
+            )
+          end
+        end
+      end
+
+      context 'with config file' do
```

Comment:
These are identical to the specs above. Copy pasta?

Reply:
Yeah... Gonna fix it. Also... I realize that hyrax is just using the chipmunk file to get by but it shouldn't be.

Reply:
FIxed that but also that lead to redoing some of solr file settings in the solr.pp and then decided to add a full hyrax test file. So working on the kinks in that.